packages/@aws-cdk/aws-servicecatalogappregistry/lib/private/util.ts

```diff
@@ -0,0 +1,10 @@
+import * as crypto from 'crypto';
```

Comment:
Let's call this file `hashing.ts`, or something. I just hate "util" files and classes. They tend to accrue garbage as time goes on.

Reply:
That sounds about right, this was similar to `servicecatalog` but I do not think we will have any other functions in here for the near future, will change.

Reply:
actually, I will just move this into application since this won't be called for attribute group or any other constructs for time being